schema/artist/index.js

```diff
@@ -462,13 +472,15 @@ const ArtistType = new GraphQLObjectType({
               is_reference: true,
               solo_show: false,
               at_a_fair: false,
+              visible_to_public: false,
               size: options.size,
             }),
             // Fair booths
             gravity('related/shows', {
               artist_id: id,
               sort: '-end_at',
               at_a_fair: true,
+              visible_to_public: false,
```

Comment:
Yes :(   This was all me btw, I take full responsibility.

The reason for this is b/c the rule we came up for 'exhibition highlights' for an artist is a fairly complex series of waterfalls (solo vs group, top tier vs lower tier, institutional vs gallery, etc.) so making all those different requests was the only solution to accomplish that.

Something that might be better, is to move that logic/waterfall upstream to Gravity, and create a synthetic score based on that, so one request with a sort on that field can return the 'highlights'